prompt/cicd.go

```diff
@@ -19,7 +19,9 @@ func PromptCICD() {
 	for len(selectedOptions) == 0 {
 		selectedOptions, _ = p.PromptMultiSelect()
 	}
-	environment := PromptEnvironment()
+	// Uncomment the below line if we environment prompt.
+	// environment := PromptEnvironment()
+	environment := constants.Development
```

Comment:
We dont need even for development. The CD file that is generated has dev, qa and master. It is kind of a generic one at this point of time

Reply:
I think it is not using anywhere. I added this because we are sending environment as arguments in functions related to that.. Otherwise I have to change all the functions which has environment as a parameter. It will work as what we expect.